TileStache/Vector/__init__.py

```diff
@@ -460,7 +461,7 @@ def _open_layer(driver_name, parameters, dirpath):
     #
     return layer, datasource
 
-def _get_features(coord, properties, projection, layer, clipped, projected, spacing, id_property):
+def _get_features(coord, properties, projection, layer, clipped, projected, spacing, id_property, empty_fields):
```

Comment:
Make this parameter optional (ie `empty_fields=True`), to keep backward compatibility